packages/desktop-client/src/components/reports/Charts.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+
+import * as monthUtils from 'loot-core/src/shared/months';
+import { integerToCurrency } from 'loot-core/src/shared/util';
+import { View, Text, Block, Button } from 'loot-design/src/components/common';
+import { colors, styles } from 'loot-design/src/style';
+
+const fontWeight = 600;
+
+export function ChartItem({
+  Icon,
+  title,
+  style,
+  handleClick,
+  bold,
+  header,
+  id,
+  reportPage,
+}) {
+  const hoverStyle = {
+    color: colors.b4,
+    cursor: 'pointer',
+    backgroundColor: 'inherit',
+  };
+  const linkStyle = [
+    header && styles.largeText,
+    {
+      color: colors.n7,
+      fontWeight: bold ? fontWeight : null,
+    },
+    { ':hover': hoverStyle },
+  ];
+
+  const content = (
+    <View
+      style={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        height: 16,
+        color: reportPage === id ? colors.n3 : 'inherit',
+      }}
+    >
+      {Icon && (
+        <Icon id={id} width={12} height={12} style={{ color: 'inherit' }} />
+      )}
+      <Block id={id} style={{ marginLeft: Icon ? 3 : 0, color: 'inherit' }}>
+        {title}
+      </Block>
+    </View>
+  );
+
+  return (
+    <View style={[{ flexShrink: 0 }, style]}>
+      <Button
+        id={title + 'Button'}
+        style={linkStyle}
+        onClick={() => handleClick(id)}
+        bare
+      >
+        {content}
+      </Button>
+    </View>
+  );
+}
+
+export function ChartExtraColumn({
+  start,
+  end,
+  totalExpenses,
+  totalIncome,
+  selectList,
+}) {
+  let amt =
+    selectList === 'Expense'
+      ? totalExpenses
+      : selectList === 'Income'
+      ? totalIncome
+      : totalExpenses + totalIncome;
+  let net = totalExpenses > totalIncome ? 'EXPENSE' : 'INCOME';
+  return (
+    <View
+      style={{
+        padding: 15,
+        overflow: 'auto',
+        flexDirection: 'column',
+        flex: 1,
+        marginTop: 15,
+        marginRight: 30,
+        marginLeft: 10,
+      }}
+    >
+      <View
+        style={{
+          backgroundColor: colors.n11,
+          height: 100,
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}
+      >
+        <Text
+          style={[
+            styles.largeText,
+            {
+              alignItems: 'center',
+              marginBottom: 2,
+              fontWeight: 600,
+            },
+          ]}
+        >
+          {monthUtils.format(start, 'MMM yyyy')} -{' '}
+          {monthUtils.format(end, 'MMM yyyy')}
+        </Text>
+      </View>
```

Comment:
Yes, it is a repeat of the months from the drop downs in the Headers. The reason I thought it relevant was so that your eye doesn't have to ping from header to the spending data in order to remember what date range you're looking at. The "Spending Total for this time period" is grouped with the time period itself. 

When reading "Total spending for this time period" I found I was asking myself "what's the time period here?" then having to look up at the header to remember then saying "okay this is total spending for Oct 2022 - Feb 2023". Just takes away all that thinking and jumping - making really clear.